test/services/EnglishUIBuilderSpec.scala

```diff
@@ -372,6 +372,22 @@ class EnglishUIBuilderSpec extends BaseSpec with ProcessJson with EnglishLanguag
     val emptyNoteGroup = NoteGroup(Seq.empty)
   }
 
+  trait WarningTest extends Test {
```

Comment:
I think code is cleaner if you create trait within tests like the Details Component and Date Input tests